src/solvers/dgmulti/dg.jl

```diff
@@ -327,6 +332,7 @@ function calc_volume_integral!(du, u, mesh::DGMultiMesh{NDIMS, <:NonAffine},
     @threaded for e in eachelement(mesh, dg, cache)
         flux_values = cache.flux_threaded[Threads.threadid()]
         for i in eachdim(mesh)
+            # Here, the broadcasting operation does not allocate
```

Comment:
Did you check this?

Reply:
Yes, there are examples which failed in the previous run ( https://github.com/trixi-framework/Trixi.jl/actions/runs/6407816141 ) which use this. For instance `elixir_euler_curved.jl (Triangular elements, Polynomial, weak formulation)` with summary CB: https://github.com/trixi-framework/Trixi.jl/actions/runs/6415628240/job/17417850521#step:7:8560

Also `elixir_navierstokes_convergence_curved.jl` uses this, with summary CB: https://github.com/trixi-framework/Trixi.jl/actions/runs/6415628240/job/17417850686#step:7:3544 and https://github.com/trixi-framework/Trixi.jl/actions/runs/6415628240/job/17417850686#step:7:7892